packages/start/config/server-handler.js

```diff
@@ -89,7 +89,17 @@ async function handleServerFunction(event) {
     contentType.startsWith("multipart/form-data") ||
     contentType.startsWith("application/x-www-form-urlencoded")
   ) {
-    parsed.push(await readFormData(event));
+    // Temporary workaround until https://github.com/unjs/nitro/issues/1721 is resolved
+    // parsed.push(await readFormData(event));
+
+    const request = new Request(getRequestURL(event), {
+      // @ts-ignore Undici option
+      duplex: "half",
```

Comment:
This is required?

Reply:
I believe so yeah. It is taken from the h3 code [here](https://github.com/unjs/h3/blob/53703dc860f1ff6fe7ce71d543deff1cfa810b11/src/utils/request.ts#L181C22-L181C22) and thought it best to leave it in.

Reply:
Seems to be related to this line in the [undici code](https://github.com/nodejs/undici/blob/6a04edcd5867c4de6bd53d10f6b44eaecf02755e/lib/fetch/request.js#L505)

Reply:
Actually testing it on a node server it seems fine, will remove it.